src/components/User/ProjectsContainer.vue

```diff
@@ -12,7 +12,8 @@ defineProps<Props>()
 <template>
   <section :class="$style.section">
     <h2 :class="$style.title">参加プロジェクト</h2>
-    <project-list :projects="projects" />
+    <project-list v-if="projects.length > 0" :projects="projects" />
+    <p v-else>これまでに参加プロジェクトはありません</p>
```

Comment:
参加したの「した」を抜く場合はこうなりそうです！
```suggestion
    <p v-else>これまでの参加プロジェクトはありません</p>
```

Reply:
「した」が抜けてたので元の提案に合わせました